tests/e2e/blocks/carousel-header.spec.js

```diff
@@ -74,6 +74,7 @@ test('Create and check carousel header block', async ({page, admin, editor}) =>
   expect(paragraphDescription1).toBe('Testing carousel description 1');
   await expect(ctaButton1).toBeVisible();
 
+  await page.waitForSelector('button.carousel-control-next');
```

Comment:
Do we really need this? The `click` function should already include waiting for the element to be visible and enabled